done/static/css/done.css

```diff
@@ -94,4 +94,34 @@
 
 .done_nogrow {
     flex-grow: 0;
-}
\ No newline at end of file
+}
+
+/* Right to left (rtl) language css */
+
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-inner {
```

Comment:
Is it possible to use bi-app here the same way that we do in platform? Then you don't need to implement styling for right-to-left languages, as every function you use is automatically language aware. I haven't tried to do this in a separate XBlock repo, so maybe that isn't possible.